rmgpy/chemkin.pyx

```diff
@@ -1795,6 +1825,12 @@ def write_kinetics_entry(reaction, species_list, verbose=True, java_library=Fals
             kinetics.n.value_si,
             kinetics.Ea.value_si / 4184.
         )
+        if isinstance(kinetics, _kinetics.SurfaceArrhenius) and kinetics.coverage_dependence:
+            if coverage_dependence is True:
+                for species, cov_params in kinetics.coverage_dependence.items():
+                    label = get_species_identifier(species)
+                    string += f'\n    COV / {label:<41}'
+                    string += f"{cov_params['E'].value_si:<9.3e} {cov_params['m'].value:<6.3f} {cov_params['a'].value:<6.3f} /"
```

Comment:
I'm pretty sure Chemkin doesn't expect SI units here for the E parameter. Is this tested?

Reply:
What units does chemkin use? It does need either `.value` or `.value_si` for this to work

Reply:
Can you check how activation energies are written elsewhere in this file? Pretty sure this will need the same units and so hopefully could be done the same way (though I have not looked yet to see how that is)

Reply:
chemkin units are specified at the top part of the reactions, but I'm not sure how rmg decides what units to write there so I will make sure they are the same

Reply:
rmg always writes in kcal/mol

Reply:
kcal/mol sounds like si to me, but I could also be wrong

Reply:
SI is Joules. kcal is 4184 times smaller. Fixed it.